makefiles/tools/bossa.inc.mk

```diff
@@ -0,0 +1,16 @@
+export FLASHER = $(RIOTBASE)/dist/tools/bossa/bossa.sh
+
+export OFLAGS = -O binary
+export HEXFILE = $(ELFFILE:.elf=.bin)
+
+# define path to bossac binary
+export BOSSAC ?= $(RIOTBASE)/dist/tools/bossa/bossac
+
+# if we go with the default (BOSSA shipped with RIOT), we download and build
```

Comment:
The autot build of bossa doesn't work for me. I have to build it first in dist:
```shell
$ make BOARD=arduino-mkr1000 -C examples/default flash
make: Entering directory '/home/aabadie/softs/src/riot/RIOT/examples/default'
Building application "default" for "arduino-mkr1000" with MCU "samd21".
[...]
env -u CXX make -C /home/aabadie/softs/src/riot/RIOT/dist/tools/bossa
mkdir -p /home/aabadie/softs/src/riot/RIOT/dist/tools/bossa/src
git clone "https://github.com/shumatech/BOSSA" "/home/aabadie/softs/src/riot/RIOT/dist/tools/bossa/src"
Cloning into '/home/aabadie/softs/src/riot/RIOT/dist/tools/bossa/src'...
[...]
COMMON_CXXFLAGS="-std=c++11" make -C /home/aabadie/softs/src/riot/RIOT/dist/tools/bossa/src strip-bossac
CPP APPLET src/WordCopyArm.cpp
CPP COMMON src/Samba.cpp
CPP COMMON src/Flash.cpp
[...]
LD /home/aabadie/softs/src/riot/RIOT/examples/default/bin/arduino-mkr1000/bossac
STRIP /home/aabadie/softs/src/riot/RIOT/examples/default/bin/arduino-mkr1000/bossac
mv: cannot stat '/home/aabadie/softs/src/riot/RIOT/dist/tools/bossa/src/bin/bossac': No such file or directory
```
As you can see, the BOSSA Makefile creates the executable in the application directory.